rules/windows/process_creation/proc_creation_win_net_use_mount_admin_share.yml

```diff
@@ -20,7 +23,7 @@ detection:
             - '\net1.exe'
         CommandLine|contains|all:
             - ' use '
-            - '\\\*\\*$' # (Specs) If some wildcard after a backslash should be searched, the backslash has to be escaped: \\*
+            - '\\\\\*\\\*$'
```

Comment:
`- '\\\\\*\\\*$' `
If I read correctly `\\` + the char `*` + `\`+ the char `*` + `$`
should not be  `\\\\*\\*$` for `\\*\*$` with * for anything ?

Reply:
Yes, you're right. It should be `\\*\*$` I will fix it.